### DISCUSSION TITLE
How to fix dir icon ?

### DISCUSSION THREAD

**Discussion by rzksobhy78** [nvim-telescope/telescope.nvim#1570] [Q&A] [answered]
![Tue Dec  7 18:38:18 EET 2021](https://user-images.githubusercontent.com/94202082/145069774-128ae9ac-e401-4063-af07-ce996f5c42b3.png)

Why directory icons look like that any help

and is there is a way to sort directories before files in file_browser

**Comment by l-kershaw** [comment, marked as answer]
Copying my answer from your reddit post for those who don't use reddit:

> The telescope file browser defaults to the nerdfonts icon, "nf-oct-file_directory" (search [here](https://www.nerdfonts.com/cheat-sheet) to see what it should look like). If you want to change the directory icon, use the `dir_icon` option (see `:h builtin.file_browser()`).
> 
> Be aware that the file browser is in the process of being moved out of Telescope core to its own plugin [here](https://github.com/nvim-telescope/telescope-file-browser.nvim).
